lib/qa/authorities/assign_fast/generic_authority.rb

```diff
@@ -0,0 +1,58 @@
+module Qa::Authorities
+  # A wrapper around the FAST api for use with questioning_authority
+  # API documentation:
+  # http://www.oclc.org/developer/develop/web-services/fast-api/assign-fast.en.html
+  class AssignFast::GenericAuthority < Base
+    attr_reader :subauthority
+    def initialize(subauthority)
+      @subauthority = subauthority
+    end
+
+    include WebServiceBase
+
+    # Search the FAST api
+    #
+    # @param [String] the query
+    # @return json results
+    def search q
+      begin
+        @raw_response = get_json(build_query_url(q))
+      rescue JSON::ParserError => error
+        # TODO: log the url and the error
```

Comment:
Attempting to write an issue appears to be leading me to just do it now. However, I do think there should be more dragons.